auth.py

```diff
@@ -53,7 +53,7 @@ def signup():
         new_user = User(email, name, hashed_password, None, None)
         create_user(new_user)
 
-        return redirect(url_for('auth.login'))
+        return redirect(url_for('root'))
```

Comment:
note that we don't login a user after signup, the user still has to login after signup

Reply:
@hongshaoyang yeah, I know, the reason for this is explained in the comment above. I'm still trying to figure it out. Will update the PR once I do. But I just figured I'd open this PR in case anybody else knows how to do it.